### ISSUE TITLE
Store latitude and longitude of events

### ISSUE THREAD

**Issue by AlbertoPdRF** [AlbertoPdRF/tisn.app#281]
**Is your feature request related to a problem? Please describe.**

Right now, we only store the country and region where an event will happen, and the creator of the event and/or its attendants have to define/agree on where to meet exactly. Having an exact point on the map indicating where the event is would be way more convenient.

**Describe the solution you'd like**

I'd like for us to store the latitude and longitude where an event will happen, as that won't only solve the problem described above, but also open a world of possible features to add. For instance, we could display events on a map instead of having a grid of cards (see the suggestion on #21). If we also stored the latitude and longitude of users, for which we could use [the Geolocation API](https://w3c.github.io/geolocation-api/), we could for example recommend events to users based on how close they are and/or send reminders on when/how to go to an event (note that some of these would require also storing the latitude and longitude of users). And these are just some quick thoughts! 😄

**Describe alternatives you've considered**

Keeping the current model.

**Additional context**

None.

**Comment by idiglove** [comment]
How are we going make the user select the longs and lats? 
Should we add a map feature to pin-point the coordinates? 

Cheers

**Comment by AlbertoPdRF** [comment]
Yep, I think adding a map feature to pin-point the coordinates would be the best way! Would you like to try to add that feature? If yes, I'll assign the issue to you!
